src/view/iView.hxx

```diff
@@ -0,0 +1,44 @@
+#ifndef I_VIEW_HXX
+#define I_VIEW_HXX
+
+#include <memory>
+
+class iRenderer;
+class Rectangle;
+class GameService;
+
+class iView
```

Comment:
Please consider putting a few comments regarding the interface.

Reply:
Agreed, it’s hard to understand what those classes are for without comments